sw/host/opentitantool/src/main.rs

```diff
@@ -29,6 +34,9 @@ enum RootCommandHierarchy {
 
 #[derive(Debug, StructOpt)]
 struct Opts {
+    #[structopt(long, default_value = "opentitantoolrc")]
```

Comment:
I don't think so: the filename given is relative to `$XDG_CONFIG_HOME` rather than `$HOME`.  That means the filename, after joining with the result of `config_dir()`, will be something like `$HOME/.config/opentitantool/<whatevs>`.

Reply:
Got it.